data/telegraf_plugins.yml

```diff
@@ -1279,6 +1286,13 @@ input:
     introduced: 0.10.3
     tags: [linux, macos, windows, servers, web]
 
+- name: Redfish
+  id: redfish
+  description: |
+    The Redfish input plugin gathers metrics and status information of hardware servers for which [DMTF's Redfish](https://redfish.dmtf.org/) is enabled.
+  introduced: 1.15.0
+  tags: [linux, macos, windows, servers, networking]
+
```

Comment:
The YAML isn't valid. Looks like the entire Redfish section needs to be indented one more level.